graphql/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "console-graphql",
+  "version": "0.0.1",
+  "main": "index.ts",
+  "license": "Apache-2.0",
+  "scripts": {
+    "dev": "nodemon --watch 'src/**/*.ts' --exec 'ts-node' src/index.ts",
+    "build": "NODE_ENV=production tsc"
+  },
```

Comment:
It may be a good idea to add a `README.md` file, with the basics of running, contributing, using.  I dont think we have to document all endpoints, etc but a quick start is prob nice.